ttnn/cpp/ttnn/operations/pool/upsample/device/upsample_program_factory_multicore.cpp

```diff
@@ -155,12 +277,12 @@ operation::ProgramWithCallbacks upsample_multi_core(
     uint32_t writer_nargs = 7;
     std::vector<uint32_t> writer_rt_args(writer_nargs);
     writer_rt_args[0] = input_stick_nbytes;
-    writer_rt_args[1] = input_nsticks_per_core / in_w;
+    writer_rt_args[1] = input_nsticks_per_core;
     writer_rt_args[2] = scale_factor_h;
     writer_rt_args[3] = scale_factor_w;
-    writer_rt_args[4] = in_w;
-    writer_rt_args[5] = out_w;
-    writer_rt_args[6] = 0;  // set for each core below
+    writer_rt_args[4] = input_nsticks_per_core;
```

Comment:
These are still referred to as in_w and out_w in the kernel. Should the first one be removed (since it's already entry [1]) and the second one renamed in the kernel?
Are there other users of this kernel?

Reply:
Removed unnecessary stuff.